facdb/sql/nysoasas_programs.sql

```diff
@@ -14,8 +14,8 @@ SELECT
     NULL as bin,
     NULL as bbl,
     service_type as factype,
-    'Chemical Dependency' as facsubgrp,
-    'NYS Office of Alcoholism and Substance Abuse Services' as opname,
+    'Substance Use Disorder Treatment Programs' as facsubgrp,
+    provider_name as opname,
```

Comment:
Do we need to implement an `initcap` function here?  That's the only question I have / thing that stands out.  Everything else looks good.